scripts/tests/twister_blackbox/test_verbose.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Blackbox tests for twister's command line, --verbose flag
+"""
+
+import importlib
+import mock
+import os
+import pytest
+import sys
+import re
+
+from conftest import ZEPHYR_BASE, testsuite_filename_mock
+from twisterlib.testplan import TestPlan
+
+
+@mock.patch.object(TestPlan, 'TESTSUITE_FILENAME', testsuite_filename_mock)
+class TestTwisterCommandLine:
+    @classmethod
+    def setup_class(cls):
+        apath = os.path.join(ZEPHYR_BASE, 'scripts', 'twister')
+        cls.loader = importlib.machinery.SourceFileLoader('__main__', apath)
+        cls.spec = importlib.util.spec_from_loader(cls.loader.name, cls.loader)
+        cls.twister_module = importlib.util.module_from_spec(cls.spec)
+
+    @classmethod
+    def teardown_class(cls):
+        pass
+
+    def test_single_v(self, capfd):
+        args = ['-p', 'qemu_x86', '-T', 'samples/hello_world', '-v']
+
+        with mock.patch.object(sys, 'argv', [sys.argv[0]] + args), \
+            pytest.raises(SystemExit) as sys_exit:
+            self.loader.exec_module(self.twister_module)
+
+        info_regex = r'INFO\s+-\s+(\d+/\d+)\s+(\S+)\s+(\S+)\s+(\S+)\s+\((\w+)\s+([\d.]+)s\)'
```

Comment:
what this line searching for?

Reply:
Sample searched-for line when using the `-v` flag:
`INFO    - 241/482 qemu_cortex_r5            tests/posix/common/portability.posix.common.static_stack PASSED (qemu 6.284s)`

Reply:
I changed regex to work with that logic, is it correct?